### ISSUE TITLE
Show which Verify filters are applied when filter dialog is hidden

### ISSUE THREAD

**Issue by nmcharlton** [Greenstand/treetracker-admin-client#81]
In the Verify page, it's possible to apply a set of filters and hide the dialog.

At this point, the user has to remember what filters they selected, and it's easy to forget that you have applied a filter, especially when moving between tools in the Admin Panel.

This is particularly troublesome because the filter dialog is hidden by default when going back to the Verify page (see video).

https://user-images.githubusercontent.com/5558838/117187148-83732c00-add3-11eb-88a4-59c5b006f739.mov

**Comment by tranquanghuy0801** [comment]
If there are a set of applied filters, can we only set the filter dialog to be visible after the user visits again or does that need a text label to show?

**Comment by nmcharlton** [comment]
> If there are a set of applied filters, can we only set the filter dialog to be visible after the user visits again or does that need a text label to show?

@tranquanghuy0801 I think it would be a good idea to show the filter dialog if the filter has been changed from the default, but even with the default set, _Verification Status_ is _Awaiting Verification_.
Showing labels for the filters that are applied would make this clearer to the user.

I think we should show these labels all the time, even when the filter dialog is open, because they reflect the _applied_ filter, which may be different to the values in the dialog if the user has changed them and not clicked _Apply_.

**Comment by tranquanghuy0801** [comment]
@nmcharlton, can I work on this?

**Comment by nmcharlton** [comment]
@tranquanghuy0801 @rgesulfo has been working on the UI design for the new suite of Admin Panel tools, including filters. Can you coordinate on this issue to ensure a consistent implementation?

**Comment by rgesulfo** [comment]
> @rgesulfo could you please also give me the permission to check this Figam file?

Of course. Sorry about that. You should be good now :)

**Comment by nmcharlton** [comment]
> @nmcharlton If we use the badge to show the current applied filters, do we need to display the list of the applied filters anymore? I reckon we should not.

@tranquanghuy0801 You're right. The badge provides the necessary visual cue for the user to check the filters.
